week-2/G-conditionals-2/exercise-3.js

```diff
@@ -9,7 +9,15 @@
 */
 
 function calculateGrade(mark) {
-
+  if (mark >= 80) {
+    return "A";
+  } else if (mark >= 60 && mark <= 80) {
```

Comment:
If the mark is 80 that would mean that it can be an A and a B, therefore is it <= or <. Same for the condition for between 50 and 60